libtiledbvcf/src/stats/variant_stats.cc

```diff
@@ -213,6 +216,10 @@ void VariantStats::init(std::shared_ptr<Context> ctx, const Group& group) {
     throw std::runtime_error(
         "encountered variant stats array version out of range while writing");
 
+  // Check for presence of "an" attribute
+  auto schema = fetch_version.schema();
+  an_present_ = schema.has_attribute("an");
```

Comment:
Why is this this checking the presence of a field, rather than using `array_version_`?